app/code/Magento/Bundle/view/frontend/templates/sales/order/shipment/items/renderer.phtml

```diff
@@ -5,7 +5,12 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */ ?>
+<?php
```

Comment:
Please remove not needed opening and closing PHP tags

Reply:
fixed